README.md

```diff
@@ -110,3 +113,30 @@ These **environment variables** should be set:
 | FONTS_URL      | <secret-URL> |
 | PORT           | 8000         |
 | PYTHON_VERSION | 3.11.4       |
+
+
+## Content Licencing
+
+Texts and content available as [CC BY](https://creativecommons.org/licenses/by/3.0/de/). 
```

Comment:
@marcoka2000 is this true? Was it agreed/validated by SenWiEnBe and CH?

Reply:
Will mail CH about that, he probably knows more.

Reply:
Ah, perfect - just wanted to do the same.
Confirmation just came in.